modules/UI/UI.js

```diff
@@ -236,6 +236,13 @@ UI.toggleFilmstrip = function() {
     APP.store.dispatch(setFilmstripVisible(!visible));
 };
 
+/**
+ * Resize film strip
+ */
+UI.resizeFilmStrip = function(width) {
```

Comment:
LEt's avoid adding more logic to this deprecated file. Please just dispatch the action inline in API.js